mummer/4.0.0-gnuplot/Dockerfile

```diff
@@ -0,0 +1,130 @@
+FROM ubuntu:focal as app
+
+# ARG sets environment variables during the build stage; they do not persist after the image is built
+ARG MUMMER_VER="4.0.0"
+ARG MASH_VER="2.3"
+
+LABEL base.image="ubuntu:focal"
+LABEL dockerfile.version="2"
```

Comment:
I would recommend resetting dockerfile.version to 1 since this PR creates a new dockerfile instead of editing an existing one

Reply:
Sorry about that! It'd fixed now.